backend/danswer/configs/app_configs.py

```diff
@@ -62,18 +58,21 @@
 )
 
 # OAuth Login Flow
-ENABLE_OAUTH = os.environ.get("ENABLE_OAUTH", "").lower() != "false"
-OAUTH_TYPE = os.environ.get("OAUTH_TYPE", "google").lower()
 OAUTH_CLIENT_ID = os.environ.get(
-    "OAUTH_CLIENT_ID", os.environ.get("GOOGLE_OAUTH_CLIENT_ID", "")
+    "OAUTH_CLIENT_ID", os.environ.get("OAUTH_CLIENT_ID", "")
```

Comment:
breaking change :( can we keep things as is?

Reply:
Or at the very least tag everyone and tell them that this is changed

Reply:
actually I'm in favor of changing this back to Google OAuth :P We use a google specific library anyhow for this.